crates/kopuz/assets/tailwind.css

```
@@ -1801,6 +1823,10 @@
   .text-\[15px\] {
     font-size: 15px;
   }
+  .leading-\[1\.1\] {
+    --tw-leading: 1.1;
+    line-height: 1.1;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)